### ISSUE TITLE
The training code of soundstream

### ISSUE THREAD

**Issue by yangdongchao** [lucidrains/audiolm-pytorch#16]
Hi, when I try to train the soundstream using your code, but I find the loss is increased. from 10 to 100, 1000,.....  and so on.
Whether you have competely train a soundstream model using this code?

**Comment by lucidrains** [comment]
@yangdongchao if there is a `wip` flag on one of my repositories, it means it isn't ready yet

i think the repository should be ready by next week's end

**Comment by lucidrains** [comment]
@yangdongchao feel free to contribute to the repository, if you can get soundstream training working on your end :pray:

**Comment by zjj1333** [comment]
Hi, I see that your wip logo is gone. Can you train the soundstream (loss drops normally) and achieve good results?

**Comment by lucidrains** [comment]
done and reportedly working
